source/parser.hera

```diff
@@ -5349,6 +5349,9 @@ TypeDeclaration
   ( Export _? )? ( Declare _? )  TypeLexicalDeclaration -> { ts: true, children: $0 }
   ( Export _? )? ( Declare _? )? TypeDeclarationRest    -> { ts: true, children: $0 }
 
+  # NOTE: TS Exported classes can have decorators before export but not after
+  Decorators ( Export _? ) !Decorators ClassDeclaration -> { ts: true, children: $0 }
```

Comment:
I don't think this should be `ts: true`. Exported classes should be in JS code too, just like other decorated classes.

Related, why is this here and not in `ExportDeclaration`?

Ah, I see, this type of decorating is inconsistent with the [TC39 decorator proposal](https://github.com/tc39/proposal-decorators)...